happiness_app/templates/happiness_app/our_team.html

```diff
@@ -4,83 +4,157 @@
+      <div class="card" style="width: 18rem;">
+        <img src="https://ca.slack-edge.com/T0L30B202-U050ACTDD41-cbad96001cef-512" class="card-img-top"
+          alt="profil_image">
+        <div class="card-body">
+          <h5 class="card-title">Stephen Dawson</h5>
+          <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's
+            content.</p>
+          <div class="container text-center">
+            <div class="row">
+              <div class="col">
+                <a href="https://github.com/stephendawsondev" class="btn btn-primary">GitHub</a>
+              </div>
+              <div class="col">
+                <a href="https://www.linkedin.com/in/sdstephendawson/" class="btn btn-primary">Linkedin</a>
+              </div>
+            </div>
+          </div>
+        </div>
       </div>
-    </div>
 
-    <div class="card" style="width: 18rem;">
-      <img src="https://avatars.githubusercontent.com/u/115192139?v=4" class="card-img-top" alt="profil_image">
-      <div class="card-body">
-        <h5 class="card-title">Hilla Muraja</h5>
-        <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
-        <a href="https://github.com/HMuraja" class="btn btn-primary">GitHub</a>
-        <a href="https://avatars.githubusercontent.com/u/115192139?v=4" class="btn btn-primary">Linkedin</a>
+      <div class="card" style="width: 18rem;">
+        <img src="https://avatars.githubusercontent.com/u/115192139?v=4" class="card-img-top" alt="profil_image">
+        <div class="card-body">
+          <h5 class="card-title">Hilla Muraja</h5>
+          <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's
+            content.</p>
+          <div class="container text-center">
+            <div class="row">
+              <div class="col">
+                <a href="https://github.com/HMuraja" class="btn btn-primary">GitHub</a>
+              </div>
+              <div class="col">
+                <a href="https://avatars.githubusercontent.com/u/115192139?v=4" class="btn btn-primary">Linkedin</a>
+              </div>
+            </div>
+          </div>
+        </div>
       </div>
-    </div>
 
-    <div class="card" style="width: 18rem;">
-      <img src="https://ca.slack-edge.com/T0L30B202-U04LMHHNQAD-1cb37693c1ce-512" class="card-img-top" alt="profil_image">
-      <div class="card-body">
-        <h5 class="card-title">Felipe Zanetti</h5>
-        <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
-        <a href="" class="btn btn-primary">GitHub</a>
-        <a href="" class="btn btn-primary">Linkedin</a>
+      <div class="card" style="width: 18rem;">
+        <img src="https://ca.slack-edge.com/T0L30B202-U04LMHHNQAD-1cb37693c1ce-512" class="card-img-top"
+          alt="profil_image">
```

Comment:
Suggestion (non-blocking): For the `alt` tag, we're expecting it in a human-readable format. So instead of "profil_image", it should be "Profile image", or even better "Felipe Zanetti's profile image" so that they are distinguishable when the screen reader is reading them aloud.